packages/react-components/src/components/table-output-component.tsx

```diff
@@ -491,6 +492,7 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
     }
```

Comment:
I think it would be better to set the tableSize in this method just before calling modelToRow(), as it doesn't really belong in that method.